src/app/viewing/responses-list/responses-list.component.css

```
@@ -0,0 +1,54 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+.response-list-card {
+    background-color: transparent;
+    position: fixed;
+    width: 16em;
+    z-index: 1;
+}
+
+.response-list-card-relative {
+  background-color: transparent;
+  position: relative;
+  width: auto;
+}
+
+#author-list {
+  height: 10em;
+  overflow-y: scroll;
+}
+
+#author-list-relative-list {
+  height: 5em;
+  overflow-y: scroll;
+}
+
+.profile-pic {
+    background: grey;
+    height: 2.5em;
+    width: 2.5em;
+    border-radius: 100%;
+}
+
+.username {
+    font-weight: bold;
+}
+
+.response-title {
+    font-size: 1.2em;
+    font-weight: bold;
+}
+
+.user-info {
+    word-wrap: break-word;
+}
+
+.hamburger-menu {
+  height: auto;
+  width: auto;
+
+}
+
+::-webkit-scrollbar {
+  width: 0px;  /* Remove scrollbar space */
```

Remove unnecessary units
```suggestion
  width: 0;  /* Remove scrollbar space */
```